gapit trim: `panic: runtime error: index out of range`

Calling `gapit trim com.graphicsfuzz.vkworker.gfxtrace` leads to a panic.

[com.graphicsfuzz.vkworker.gfxtrace.zip](https://github.com/google/gapid/files/2869870/com.graphicsfuzz.vkworker.gfxtrace.zip)

```
panic: runtime error: index out of range [recovered]
	panic: runtime error: index out of range

goroutine 1 [running]:
github.com/google/gapid/core/app/crash.Crash(0xf6f6a0, 0x177be30)
	core/app/crash/crash.go:89 +0xe1
github.com/google/gapid/core/app.Run.func1()
	core/app/run.go:161 +0xec
panic(0xf6f6a0, 0x177be30)
	GOROOT/src/runtime/panic.go:513 +0x1b9
main.(*trimVerb).getDCERequest(0xc00021b560, 0xc00011d0c0, 0x5, 0x8, 0xc0001dc2d0, 0x0, 0x0, 0x0)
	cmd/gapit/trim.go:121 +0x70b
main.(*trimVerb).Run(0xc00021b560, 0x115b5a0, 0xc0002550b0, 0xc00027e220, 0x0, 0x0, 0x1, 0x0, 0xc000254db0, 0xc00027e210, ...)
	cmd/gapit/trim.go:58 +0x5cb
github.com/google/gapid/core/app.(*Verb).Invoke(0x17d7500, 0x115b5a0, 0xc0002550b0, 0xc00027e200, 0x2, 0x2, 0x0, 0x0)
	core/app/verbs.go:92 +0x742
github.com/google/gapid/core/app.VerbMain(0x115b5a0, 0xc0002550b0, 0x0, 0x0)
	core/app/verbs.go:123 +0x9a
github.com/google/gapid/core/app.Run(0x1097da0)
	core/app/run.go:253 +0xbc5
main.main()
	cmd/gapit/main.go:24 +0x7d
```

--- a/cmd/gapit/trim.go
+++ b/cmd/gapit/trim.go
@@ -114,7 +114,7 @@ func (verb *trimVerb) eofEvents(ctx context.Context, capture *path.Capture, clie
 func (verb *trimVerb) getDCERequest(eofEvents []*service.Event, p *path.Capture) []*path.Command {
 	frameCount := verb.Frames.Count
 	if frameCount < 0 {
-		frameCount = len(eofEvents) - verb.Frames.Start + 1
+		frameCount = len(eofEvents) - verb.Frames.Start
 	}
 	dceRequest := make([]*path.Command, 0, frameCount+len(verb.ExtraCommands))
 	for i := 0; i < frameCount; i++ {

